docs/apps/guacamole.md

```diff
@@ -1 +1,64 @@
-Placeholder page
+# This guide will help you follow Gilbn's tutorial to protect your Guacamole install with F2B.
+
+Since DockSTARTer uses Oznu's image for Guacamole, it only generates logs inside the container itself. Following these steps will allow you to get the Guacamole container to generate a log file in `~/.config/appdata/guacamole` which you can then mount to the LetsEncrypt container so F2B can monitor it and ban malicious IPs.
+
+You can find Gilbn's tutorial [here](https://technicalramblings.com/blog/remotely-accessing-the-unraid-gui-with-guacamole-and-vnc-web-browser/). You will need to follow it after completing the following steps:
+
+1. Create a `logback.xml` file inside `~/.config/appdata/guacamole/guacamole`
+    1. `touch ~/config/appdata/guacamole/guacamole`
+  <br> or
+    1. `sudo nano ~/.config/appdata/guacamole/guacamole/logback.xml`
+1. Open the file with your favorite editor and place the following contents inside of it: 
+ <br> **NOTE: Make sure to make changes to the timezone accordingly. Check the `php-local.ini` file in `~/.config/appdata/letsencrypt/php` if you are not sure what your timezone is.** 
+ ```
+ <configuration>
+        <!-- Appender for debugging -->
+        <appender name="GUAC-DEBUG" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder>
+                        <pattern>%d{HH:mm:ss.SSS, America/New_York} [%thread] %-5level %logger{36} - %msg%n</pattern>
+                </encoder>
+        </appender>
+        <!-- Appender for debugging in a file-->
+        <appender name="GUAC-DEBUG_FILE" class="ch.qos.logback.core.FileAppender">
+                <file>/usr/local/tomcat/logs/guacd.log</file>
```

Comment:
Why not use `/config/guacamole/guacd.log` ? then  you would not have to worry about a separate volume on the guac container. 

You still need the volume on the letsencrypt container though, which you have below, but you do not have the volume on the guac container in your override instructions below so...

Reply:
Wouldn't this mean you have to mount two different volumes in the overrides file then? One for letsencrypt and another for guac.

Reply:
if your file is at `/usr/local/tomcat/logs/guacd.log` then yes you would need a volume in your override for this location on guac and a volume for letsencrypt to see the file where you need it. If it is at `/config/guacamole/guacd.log` then you should only need to add a volume in your override for letsencrypt.

Reply:
Cool, fixed.